BFAIR/FIA_MS/import_statistics.py

```diff
@@ -0,0 +1,123 @@
+import numpy as np
+import pandas as pd
+from pyopenms import FeatureMap, FeatureXMLFile
+
+
+def extractNamesAndIntensities(feature_dir, sample_names, database):
+    """
+    This function take .featureXML files, the output of SmartPeak
```

Comment:
Does this format conform to the Numpy Docstrings https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard?

Reply:
If you don't mind double checking for typos in the description as well that will aid later users.